frontend/login_menu.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+// ======================================== Logowanie do strony ======================================== //
```

Comment:
Comments this long are not more helpful than normal comments, and can even be more annoying on smaller screens or when the code part of IDE is smaller in width than your editor.
In other words, don't use the equal signs to make the comment "nicer"